examples/newrelic-express-apm/src/controller/ProductController.js

```diff
@@ -10,16 +11,25 @@ class ProductController {
   }
 
   async getProducts(req, res) {
+    const start = performance.now()
     const data = await this.productService.getProduct(req.body)
 
     if (data.statusCode == 200) {
+      const end = performance.now()
+      newrelic.recordMetric(
+        'Commercetools/Test/Response/Time/Total',
+        end - start
+      )
       return ResponseHandler.successResponse(
         res,
         data.statusCode || data.body.statusCode,
         data.message || data.body.message,
         data.body
       )
     }
+
+    const end = performance.now()
+    newrelic.recordMetric('Commercetools/Test/Response/Time/Total', end - start)
```

Comment:
You have duplicated code.

btw. should this be recorded inside the ts-apm-sdk? There is a metric `Commercetools/Client/Response/Total` already. Why would the user need the second metric here?

Reply:
Uhmmm not necessarily a duplicate. What I was looking at is how the custom can instrument an individual function in their codebase. As you may have seen, the only thing we recorded in the `telemetryMiddleware` was the response time. However with this, they can record any other metric include a status code, execution time of a function etc.

Reply:
OK, I see your point. I find it a bit weird that we offer also newrelic, datadog etc. when the customer could actually use the original newrelic or datadog packages by themselves. 
We do not add anything extra to it, so I'm wondering why would the customer wants to use newrelic from `ts-sdk-apm` package if they just want to use newrelic for their custom metrics? Would it be better for them to use directly the package from newrelic directly?